Source/Core/Core/SlippiTimer.cpp

```diff
@@ -6,20 +6,17 @@ extern std::unique_ptr<SlippiPlaybackStatus> g_playback_status;
 
 void slippiTimer::Notify()
 {
-	int totalSeconds = (int) ((g_playback_status->latestFrame + 123) / 60);
-	int totalMinutes = (int)(totalSeconds / 60);
-	int totalRemainder = (int)(totalSeconds % 60);
+	unsigned int totalSeconds = (g_playback_status->latestFrame + 123) / 60;
+	unsigned char totalMinutes = totalSeconds / 60;
+	unsigned char totalRemainder = totalSeconds % 60;
 
-	int currSeconds = int ((g_playback_status->currentPlaybackFrame + 123) / 60);
-	int currMinutes = (int)(currSeconds / 60);
-	int currRemainder = (int)(currSeconds % 60);
-	// Position string (i.e. MM:SS)
-	char endTime[5];
-	sprintf(endTime, "%02d:%02d", totalMinutes, totalRemainder);
-	char currTime[5];
-	sprintf(currTime, "%02d:%02d", currMinutes, currRemainder);
+	unsigned int currSeconds = (g_playback_status->currentPlaybackFrame + 123) / 60;
+	unsigned char currMinutes = currSeconds / 60;
+	unsigned char currRemainder = currSeconds % 60;
 
-	std::string time = std::string(currTime) + " / " + std::string(endTime);
+	std::string time =
+		std::to_string(totalMinutes) + ":" + std::to_string(totalRemainder) + " / " +
+		std::to_string(currMinutes) + ":" + std::to_string(currRemainder);
```

Comment:
Isn't this not identical? Won't times show up as like 2:3 instead of 02:03 now?

Reply:
fixed both instances in latest, went too fast with this one